app/containers/ActionSheet/ActionSheet.tsx

```diff
@@ -122,69 +102,38 @@ const ActionSheet = React.memo(
 			</>
 		);
 
-		const renderFooter = () =>
-			data?.hasCancel ? (
-				<Button
-					onPress={hide}
-					style={[styles.button, { backgroundColor: themes[theme].auxiliaryBackground }]}
-					// TODO: Remove when migrate Touch
-					theme={theme}
-					accessibilityLabel={I18n.t('Cancel')}>
-					<Text style={[styles.text, { color: themes[theme].bodyText }]}>{I18n.t('Cancel')}</Text>
-				</Button>
-			) : null;
-
-		const renderItem = ({ item }: { item: IActionSheetItem['item'] }) => <Item item={item} hide={hide} />;
-
-		const animatedPosition = React.useRef(new Value(0));
-		const opacity = interpolateNode(animatedPosition.current, {
-			inputRange: [0, 1],
-			outputRange: [0, themes[theme].backdropOpacity],
-			extrapolate: Extrapolate.CLAMP
-		}) as any; // The function's return differs from the expected type of opacity, however this problem is something related to lib, maybe when updating the types will be fixed.
+		const renderBackdrop = useCallback(
+			props => (
+				<BottomSheetBackdrop
+					{...props}
+					appearsOnIndex={0}
+					// Backdrop should be visible all the time bottom sheet is open
+					disappearsOnIndex={-1}
+					opacity={colors.backdropOpacity}
+				/>
+			),
+			[]
+		);
 
 		const bottomSheet = isLandscape || isTablet ? styles.bottomSheet : {};
 
 		return (
 			<>
 				{children}
 				{isVisible && (
-					<>
-						<TapGestureHandler onHandlerStateChange={onBackdropPressed}>
-							<Animated.View
-								testID='action-sheet-backdrop'
-								style={[
-									styles.backdrop,
-									{
-										backgroundColor: themes[theme].backdropColor,
-										opacity
-									}
-								]}
-							/>
-						</TapGestureHandler>
-						<ScrollBottomSheet<TActionSheetOptionsItem>
-							testID='action-sheet'
-							ref={bottomSheetRef}
-							componentType='FlatList'
-							snapPoints={snaps}
-							initialSnapIndex={closedSnapIndex}
-							renderHandle={renderHandle}
-							onSettle={index => index === closedSnapIndex && toggleVisible()}
-							animatedPosition={animatedPosition.current}
-							containerStyle={{ ...styles.container, ...bottomSheet, backgroundColor: themes[theme].focusedBackground }}
-							animationConfig={ANIMATION_CONFIG}
-							data={data.options}
-							renderItem={renderItem}
-							keyExtractor={item => item.title}
-							style={{ backgroundColor: themes[theme].focusedBackground }}
-							contentContainerStyle={styles.content}
-							ItemSeparatorComponent={List.Separator}
-							ListHeaderComponent={List.Separator}
-							ListFooterComponent={renderFooter}
-							getItemLayout={getItemLayout}
-							removeClippedSubviews={isIOS}
-						/>
-					</>
+					<BottomSheet
```

Comment:
![image](https://user-images.githubusercontent.com/804994/168337358-32d33eeb-e992-4f1b-997a-182732b614c8.png)
It should be centralized. You can test it on tablets as well.

Reply:
Done

![Screenshot_1652526699](https://user-images.githubusercontent.com/77742477/168423304-7decd8a6-1c47-4167-9af9-a948d4258e0e.png)